src/layer/marker/DivIcon.js

```diff
@@ -44,7 +44,12 @@ export var DivIcon = Icon.extend({
 		var div = (oldIcon && oldIcon.tagName === 'DIV') ? oldIcon : document.createElement('div'),
 		    options = this.options;
 
-		div.innerHTML = options.html !== false ? options.html : '';
+		if (options.html instanceof Element) {
+			div.innerHTML = '';
```

Comment:
I'd rather see `L.DomUtil.empty()` in use here.

Reply:
Sure. No problem.